### PR TITLE
Fix gameplay always skipping to first hitobject time

### PR DESCRIPTION
**Pull request by peppy**
Regresssed with previous build

- [ ] Depends on #3103.

### CODE REVIEW

**Comment by ekrctb** [comment]
This is not resolving #3043, right?

**Comment by peppy** [comment]
It doesn't, no. This fixes only the blatant regression for any map with lead-in time.

File: osu.Game/Screens/Play/Player.cs [hunk 1]
```diff
@@ -139,7 +139,7 @@ private void load(AudioManager audio, APIAccess api, OsuConfigManager config)
             adjustableClock = new DecoupleableInterpolatingFramedClock { IsCoupled = false };
 
             adjustableClock.Seek(AllowLeadIn
-                ? Math.Min(RulesetContainer.GameplayStartTime, beatmap.HitObjects.First().StartTime - beatmap.BeatmapInfo.AudioLeadIn)
+                ? Math.Min(0, beatmap.HitObjects.First().StartTime - beatmap.BeatmapInfo.AudioLeadIn)
```
Comment (smoogipoo) [inline osu.Game/Screens/Play/Player.cs]: Shouldn't the second part of this be `GameplayStartTime - AudioLeadIn`?

Comment (peppy) [reply osu.Game/Screens/Play/Player.cs]: it really should; not sure how that got merged.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**